okapi-core/src/main/raml/ModuleUser.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "LaunchDescriptor.json",
+  "title": "LaunchDescriptor",
+  "description": "Specifies the user that should be created for the module",
+  "type": [ "object", "null" ],
+  "additionalProperties" : false,
+  "properties": {
+    "type": {
+      "description": "The type of the module user",
+      "type": "string",
+      "enum": ["module", "tenant"],
+      "default": "module"
+    },
+    "permissions": {
+      "description": "Permission ID (usually module.service.method or similar)",
```

Comment:
I don't understand this description. Isn't this the permissions that the module user requires to operate?

Reply:
done